Tasks/CargoAuthenticateV0/taskJsonOverride.json

```diff
@@ -0,0 +1,25 @@
+{
+    "inputs": [
+        {
+            "name": "workloadIdentityServiceConnection",
+            "label": "'Entra Workload ID-backed Azure DevOps user' Service Connection",
+            "helpMarkDown": "If this is set, feedUrl is required. All other inputs are ignored.",
+            "type": "connectedService:workloadidentityuser",
+            "required": false,
+            "properties": {
+                "EditableOptions": "False",
+                "MultiSelectFlatList": "False"
+            },
+            "groupName": "feedAuthentication"
+        },
+        {
+            "name": "feedUrl",
+            "label": "Azure Artifacts Feeds url.",
```

Comment:
nit feeds should be singular feeds to promote customer understanding this is a single url input\